### ISSUE TITLE
Invalid Config with Blazegraph

### ISSUE THREAD

**Issue by valeviolin** [architolk/Linked-Data-Theatre#12]
I am trying to connect LDT with [Blazegraph](http://blazegraph.com) but I am getting an INVALID Config on the info page. 

Messages:

    Reason for invalid configuration:
    [001] Non of the defined sites matches the domain of the URL (expected site-domain: 139.162.197.214:8080)
    [003] Non of the defined stages matches the URL for a site at domain: 139.162.197.214:8080

and the config.xml

```
<theatre env="dev" configuration-endpoint="http://127.0.0.1:9999/sparql" local-endpoint="http://127.0.0.1:9999/sparql" sparql="yes">
	<!-- Remove the "backstage" attribute in a production environment (or secure that particular URL) -->
	<site domain="localhost" icon="favicon.ico" backstage="localhost">
		<stage/>
	</site>
	<!-- The default port of Tomcat is 8080, so the site below will give you access to a LDT on a default Tomcat install -->
	<!-- The configuration of the LDT for the stage below is expected in the named graph <http://localhost:8080/stage> -->
	<site domain="localhost:8080" icon="favicon.ico" backstage="localhost:8080">
		<stage/>
	</site>
	<!-- Optional: include a third site, with two stages and a backstage for every stage -->
	<!-- Example -->
	<!--
	<site domain="localhost:8080" backstage="localhost:8080">
		<stage name=""/>
		<stage name="stage2"/>
	</site>
	-->
	<!-- End example -->
	<subdomain/>
	<query/>
	<format/>
	<subject/>
	<term/>
	<representation/>
	<roles>
		<!-- Add your own roles for security -->
		<!-- Example -->
		<!--
		<role>admin</role>
		-->
		<!-- End example -->
	</roles>
	<!-- Example endpoint -->
	<!--
	<endpoint url="http://some-secure-endpoint/sparql">
		<username>john</username>
		<password>somesecret</password>
	</endpoint>
	-->
</theatre>
```

So basically I've only changed the port of the sparql endpoint. Both LDT and Blazegraph are in the same server.

**Comment by architolk** [comment]
The reason you're getting the invalid config, is that you are probably trying to access the LDT by the URL: `http://139.162.197.214:8080`, but in the config the domain is `localhost:8080`. So you should change the config to `139.162.197.214:8080` to resolve this error. (The LDT looks at the URL to find the correct config, and in this case: no config could be found).
